scripts/managed-resources/generate-managed-list.py

```diff
@@ -38,6 +40,51 @@ def get_api_resource_kinds():
     return [i.decode("utf-8") for i in r.splitlines()]
 
 
+def collect_ocp_release_namespaces():
+    """
+    Returns a dict of namespaces present in the manifests provided by 'ocm adm release extract'
+    """
+    ocp_namespaces = []
+    with tempfile.TemporaryDirectory() as tmpdir:
+        try:
+            r = subprocess.check_output(
+                ["oc", "adm", "release", "extract", "--to", tmpdir],
+                stderr=subprocess.DEVNULL,
+            )
+        except subprocess.CalledProcessError as e:
+            sys.exit(f"Failed to extract OCP release manifests: {e}")
+        yaml_file_paths = [
+            os.path.join(tmpdir, f) for f in os.listdir(tmpdir) if f.endswith(".yaml")
+        ]
+        for file_path in yaml_file_paths:
+            with open(file_path, "r") as f:
+                manifests = yaml.full_load_all(f)
+                for manifest in manifests:
+                    if manifest is not None and manifest["kind"] == "Namespace":
+                        ocp_namespaces.append(manifest["metadata"]["name"])
+    resources = dict()
+    resources["Resources"] = {}
+    resources["Resources"]["Namespace"] = [{"name": ns} for ns in ocp_namespaces]
```

Comment:
Any chance we can sort the array?
```suggestion
    ocp_namespaces.sort()
    resources["Resources"]["Namespace"] = [{"name": ns} for ns in ocp_namespaces]
```